release-notes.md

```diff
@@ -0,0 +1,64 @@
+# v0.1.0
+
+This release has been tracked through the [0.1.0 project](https://github.com/intel/cloud-hypervisor/projects/1).
+
+Highlights for `cloud-hypervisor` version 0.1.0 include:
+
+### Shared filesystem
+
+We added support for the [virtio-fs](https://virtio-fs.gitlab.io/) shared file
+system, allowing for an efficient and reliable way of sharing a filesystem
+between the host and the `cloud-hypervisor` guest.
+
+See our [filesystem sharing](https://github.com/intel/cloud-hypervisor/blob/master/docs/fs.md)
+documentation for more details on how to use virtio-fs with `cloud-hypervisor`.
+
+### Initial direct device assignment support
+
+VFIO (Virtual Function I/O) is a kernel framework that exposes direct device
+access to userspace. `cloud-hypervisor` uses VFIO to directly assign host
+physical devices into its guest.
+
+See our [VFIO](https://github.com/intel/cloud-hypervisor/blob/master/docs/vfio.md)
+documentation for more detail on how to directly assign host devices to
+`cloud-hypervisor` guests.
+
+### Userspace IOAPIC
+
+`cloud-hypervisor` supports a so-called split IRQ chip implementation by
+implementing support for the [IOAPIC](https://wiki.osdev.org/IOAPIC).
+By moving part of the IRQ chip implementation from kernel space to user space,
+the IRQ chip emulation does not always run in a fully privileged mode.
+
+### Fake persistent memory
+
+The `virtio-pmem` implementation emulates a fake persistent memory device that
+`cloud-hypervisor` can e.g. boot from. Booting from a `virtio-pmem` device
+allows to bypass the guest page cache and improve the guest memory footpring.
+
+### Linux kernel big Zimage (a.k.a. `bzImage`)
```

Comment:
I think you can skip big zImage and just go with bzImage

Reply:
Done, thanks.